<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>숙박종류</string>
   </property>
   <widget class="QRadioButton" name="k_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>한옥</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>리조트</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>모텔</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="p_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>펜션</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="c_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>캠핑/글램핑</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="h_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>호텔</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="g_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>게스트하우스</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>숙박형태</string>
   </property>
   <widget class="QRadioButton" name="kl_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>대실</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="l_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>숙박</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>숙소정보</string>
   </property>
   <widget class="QLineEdit" name="l_inf">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="r_inf">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>숙박번호</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>방번호</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>체크인/아웃</string>
   </property>
   <widget class="QDateEdit" name="c_out">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="c_in">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>checkIn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>checkOut</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>450</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="data_view">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>611</width>
     <height>411</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>숙박종류</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>숙박형태</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>숙박번호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>방번호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>체크인</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>체크아웃</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
